qiskit_nature/drivers/molecule.py

```diff
@@ -412,3 +412,13 @@ def perturbations(self) -> Optional[List[float]]:
     def perturbations(self, value: Optional[List[float]]) -> None:
         """Set perturbations"""
         self._perturbations = value
+
+    @property
+    def degrees_of_freedom(self) -> Optional[List[Callable]]:
+        """Get degrees_of_freedom"""
```

Comment:
Maybe you can improve these docstrings slightly. E.g.
```
Get the degrees of freedom.
```
Same for the setter

Other than that this LGTM. Thanks! :+1: